### DISCUSSION TITLE
Mercator Ikuü SPP04G reporting multiple times a second

### DISCUSSION THREAD

**Discussion by Martin-Dolan** [Koenkk/zigbee2mqtt#18602] [Devices]
I've got one of the Mercator Ikuü SPP04G 4 gang outlets.  It shows up as two devices - the left pair and right pair.  That's normal.
Controls work etc, no problem.  what is a problem is the two devices spam the absolute sh!t out of my zigbee network. 
As soon as I let them join, if you look at the log file in zigbee2mqtt,  each one reports its status at least twice a second. 
I know its good to keep things up to date , but seriously? haha

Any idea how to fix this?  I tried to play around  with the reporting interval but couldn't find anything that would make a difference.

Don't think its a faulty device as they are basically two separate 2 gang power outlets that just happen to live next door to each other and they would both have to have the same fault which is unlikely.

 Any ideas?  

ps. here is a screenshot of the log...
![zigbee_ikuu-spp04g](https://github.com/Koenkk/zigbee2mqtt/assets/11151228/b72800ea-02b4-4386-ac75-ff586388ba7d)

**Comment by JacekKac** [comment]
the same here, i have 20 [TS011F_plug_1](https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html#tuya-ts011f_plug_1) spamming all the time i want to turn off power mesaurment raporting, but playing with reporting settings doesnt change this
